### DISCUSSION TITLE
Pubblicare l'esempio dati Città Metropolitana di Napoli

### DISCUSSION THREAD

**Discussion by aborruso** [ondata/italian-public-sector-pnrr-data-guide#13] [Proposte]
### Testo

Sono qui https://dati.gov.it/view-dataset/dataset?id=13dd88c7-5d9d-4de9-9695-38378b0e969b

Mostrare come incrociarli con i dati delle fonti del vademecum a partire dal CUP.

**Comment by Magantgiu** [comment]
La pagina non esiste più

**Reply by aborruso** [reply]
> La pagina non esiste più

Ciao  e grazie. Hanno cambiato URL. Ora dovrebbe essere https://www.dati.gov.it/view-dataset/dataset?id=92c397dc-746d-42d8-9087-f19daf5c1155
